oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandlerTest.java

```diff
@@ -90,7 +90,10 @@ class AuthenticationCallbackHandlerTest {
     private static final String CLIENT_SESSION_ID = "a-client-session-id";
     private static final ClientID CLIENT_ID = new ClientID();
     private static final String CLIENT_NAME = "client-name";
-    private static final Subject INTERNAL_PAIRWISE_ID = new Subject();
+    private static final String TEST_INTERNAL_COMMON_SUBJECT_IDENTIFIER =
+            "internal-common-subject-identifier";
+    private static final Subject INTERNAL_PAIRWISE_ID =
```

Comment:
what are your thoughts about consolidating the naming conventions for this to just one name? ie rename internalPairwiseId instances to internalCommonSubjectIdentifier. I imagine we'll be touching almost every use of this subject, so it's a good opportunity to name them consistently.

Reply:
Yeah, if you spot any that I've missed let me know

Reply:
I think it makes sense to use internalCommonSubjectId instead of internalCommonSubjectIdentifier. More consistent with other ID names.

Reply:
instances of `INTERNAL_PAIRWISE_ID` should be replaced with `TEST_INTERNAL_COMMON_SUBJECT_ID`. There's a ticket in tech debt to type these as eg `InternalSubjectId`, but it can't be extended from the type `Subject` (that type is `final`), so I reckon it's ok to just use the `String` rather than the `Subject`.

Reply:
Ah, I misunderstood you, I see what you're saying